src/Groups/matrices/MatGrp.jl

```diff
@@ -75,8 +75,42 @@ MatrixGroupElem(G::MatrixGroup{RE,T}, x::T, x_gap::GapObj) where {RE,T} = Matrix
 MatrixGroupElem(G::MatrixGroup{RE,T}, x::T) where {RE, T} = MatrixGroupElem{RE,T}(G,x)
 MatrixGroupElem(G::MatrixGroup{RE,T}, x_gap::GapObj) where {RE, T} = MatrixGroupElem{RE,T}(G,x_gap)
 
+"""
+    ring_elem_type(G::MatrixGroup{S,T}) where {S,T}
+    ring_elem_type(::Type{MatrixGroup{S,T}}) where {S,T}
+
+Return the type `S` of the entries of the elements of `G`.
+One can enter the type of `G` instead of `G`.
+
+# Examples
+```jldoctest
+julia> g = GL(2, 3);
+
+julia> ring_elem_type(typeof(g)) == elem_type(typeof(base_ring(g)))
```

Comment:
So this is also the same as this then?
```suggestion
julia> ring_elem_type(typeof(g)) == elem_type(base_ring_type(g))
```
Makes me wonder if we really need `ring_elem_type`. Ah well, I guess we have it for now...

Reply:
The change from `typeof(base_ring(X))` to `base_ring_type(X)` would make sense across Oscar (perhaps for another pull request).